Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -288,6 +310,27 @@ void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef* hcan)
     }
 
     //TODO: Match StdId of header and data length content for green and blue messages and check data for set bit of toggling green and blue led
+
+    if(hdr.DLC == 1){
+        if(BLUE_MESSAGE_STDID == hdr.StdId)
+        {
+            uint32_t bitmaskBlue = 10000101;
```

Comment:
I think this ends up being the integer ten million one hundred and one. You want to use `0b10000101`
https://stackoverflow.com/a/29487281

Reply:
This mask is also incorrect. Double check which bits need to be set.